kolibri_instant_schools_plugin/assets/src/views/profile-page/index.vue

```diff
@@ -295,11 +287,8 @@
     margin-right: auto
     width: ($iphone-width - 20)px
 
-  .submit
-    margin-left: auto
-    margin-right: auto
-    display: block
-
+  button[type='submit']
```

Comment:
What should do for changes that should also be part of the main user plugin, like this change?